src/fr/franime/src/eu/kanade/tachiyomi/animeextension/fr/franime/FrAnime.kt

```diff
@@ -51,6 +51,8 @@ class FrAnime : AnimeHttpSource() {
 
     // === Anime Details
 
+    override fun fetchAnimeDetails(anime: SAnime): Observable<SAnime> = Observable.just(anime)
```

Comment:
Doesn't this mean if the anime details change on the site, we can never update the anime in our database?

Reply:
The source uses a 6Mb file holding their whole database which is then processed locally by their webpage, currently the file is downloaded once when the extension is loaded. To refresh it you just need to close and reopen the app. I think it's an acceptable tradeoff ?

Reply:
I guess we have no better choice